src/features/Ens/__tests__/EnsTable.test.tsx

```diff
@@ -3,31 +3,26 @@ import React from 'react';
 import { simpleRender } from 'test-utils';
 
 import { translateRaw } from '@translations';
+import { DomainNameRecord } from '@types';
 
 import { EnsTable } from '../EnsTable';
-import { DomainNameRecord } from '../types';
 
 /* Test components */
 describe('ENSTable', () => {
```

Comment:
Why are we testing `EnsTable` instead of `EnsDashboard` now?

Reply:
The file and the test were called EnsDashboard.test but only tested EnsTable.

Reply:
lol, no worries then 😆